src/test/java/com/ocient/jdbc/JDBCBenchmarks.java

```diff
@@ -0,0 +1,202 @@
+package com.ocient.jdbc;
+
+// Junit imports
+import static org.junit.Assert.*;
+
+// Java imports
+import java.util.Properties;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.io.File;
+import java.io.PrintStream;
+// JDBC imports
+import java.sql.Connection;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import com.ocient.jdbc.XGConnection.ConnectionProperty;
+
+// JMH imports
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+@State(Scope.Thread)
+public class JDBCBenchmarks{
+
+    public static void main(final String[] args) throws Exception {
+        // JDBCBenchmarks test = new JDBCBenchmarks();
+        // test.beforeClass();
+        // test.beforeTest();
+        // ResultSet rs = test.executeAndRetrieveUtility(
+        //     "select INT(100) as c1 from sys.dummy10"
+        // );
+
+
+        // while(rs.next()) {
+        //     System.out.println(rs.getInt("c1"));
+        // }
+
+        // m_conn.close();
+
+
+        PrintStream stream = new PrintStream(new File("/ocient/db/ocient_jdbc/test.txt"));
+        System.setOut(stream);
+
+        final Options opt =
+            new OptionsBuilder()
+                .include(JDBCBenchmarks.class.getSimpleName())
+                .forks(1)
+                .warmupIterations(2)
+                .measurementIterations(5)
+                .build();
+        Runner r = new Runner(opt);
+        r.run();
+    }
+
+    private static String m_url = "";
+
+    private static AtomicInteger teardownCounter = new AtomicInteger();
+
+    @Param({"blocking", "nonblocking"})
+    private static String m_transport;
+    private static Properties m_properties = new Properties();;
+
+    private static Connection m_conn;
+    private static Statement m_stmt;
```

Comment:
static implies global (at the class loader context w.r.t. to the JVM)
```suggestion
    private Connection m_conn;
    private Statement m_stmt;
```